### PR TITLE
Docs: Add Styletron example

### PR DESCRIPTION
**Pull request by Zhangdroid**
#513

### CODE REVIEW

File: docs/Thirdparties.md [hunk 1]
```diff
@@ -194,3 +195,46 @@ export function Button({ color, size, children }) { /* ... */ }
...
+```jsx
+const Styletron = require('styletron-client');
+const { StyletronProvider } = require('styletron-react');
+
+<StyletronProvider styletron={new Styletron()}>
+  <App greeting="Choose your pizza!"/>
+</StyletronProvider>
+```
+
+To use Styletron store in every component, which is the same way as [Redux](#redux), redefine the Wrapper component:
```
Comment (sapegin) [inline docs/Thirdparties.md]: “store“ → ””?
“which is the same way as [Redux](#redux)” → “similar to [Redux](#redux)”?

File: docs/Thirdparties.md [hunk 2]
```diff
@@ -194,3 +195,46 @@ export function Button({ color, size, children }) { /* ... */ }
...
+    Wrapper: path.join(__dirname, 'lib/styleguide/Wrapper')
+  }
+};
+```
+
+```jsx
+// lib/styleguide/Wrapper.js
+import React, { Component } from 'react';
+
+const Styletron = require('styletron-client');
```
Comment (sapegin) [inline docs/Thirdparties.md]: I’d use imports for consistency here.

File: docs/Thirdparties.md (hunk 1 above)
Comment (Zhangdroid) [reply docs/Thirdparties.md]: Sorry, it's copy from redux part and I didn't look carefully, fixed :)

File: docs/Thirdparties.md (hunk 2 above)
Comment (Zhangdroid) [reply docs/Thirdparties.md]: fixed

**Comment by sapegin** [comment]
Thanks!

**Pull request merged**